src/appkit.cs

```diff
@@ -18031,6 +18039,14 @@ public partial interface NSToolbar {
 		[Mac (10,10)]
 		[Export ("allowsExtensionItems")]
 		bool AllowsExtensionItems { get; set; }
+
+		[Mac (10,11)]
+		[Field ("NSToolbarToggleSidebarItemIdentifier")]
+		NSString NSToolbarToggleSidebarItemIdentifier { get; }
```

Comment:
I agree. I was matching the other 10 above it (that you can't see in diff easily) that also have the bad naming. :(

I figured better to match existing than fix it.